sys/arm64/vmm/arm64.h

```diff
@@ -168,6 +168,7 @@ DEFINE_VMMOPS_IFUNC(int, restore_tsc, (void *vcpui, uint64_t now))
 #endif
 #endif
 
+uintptr_t	vmm_call_hyp_init(uintcap_t);
```

Comment:
Ideally it'd be `vmm_call_hyp(HYP_INIT, address)` upstream and the stub would check that x0 is HYP_INIT, making it look more like a normal hypervisor call and removing this oddity downstream

Reply:
Yes, that would be more sensible, but I'd prefer to defer making a change like that until bhyve's landed upstream.

Reply:
Agreed